src/components/Button.tsx

```diff
@@ -0,0 +1,38 @@
+import { ButtonHTMLAttributes, ReactNode } from 'react';
+
+interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children?: ReactNode;
+  onClick?: () => void;
+  size?: 'sm' | 'md' | 'lg';
+  color?: string;
+}
+
+function Button({
+  children,
+  onClick,
+  type,
+  size = 'md',
+  color = 'default',
+}: Props) {
+  const sizeStyle =
+    size === 'sm'
+      ? 'px-14pxr py-8pxr text-12pxr'
+      : size === 'md'
+        ? ' px-16pxr py-10pxr text-14pxr'
+        : 'px-20pxr py-16pxr text-16pxr';
```

Comment:
사이즈에 대한 타입이 `'sm' | 'md' | 'lg'` 지금의 3종류보다 늘어날 것을 고려하여 이중 삼항연산자보다는 객체로 관리하는 것도 좋아보이네요. (당장, 3종류에 대해서도 가독성이 아쉬워 보여서요!)

```suggestion
// sizeStyles 은 Button 밖에 선언하는 것이 더 좋습니다.
const sizeStyles = {
  sm: 'px-14pxr py-8pxr text-12pxr',
  md: 'px-16pxr py-10pxr text-14pxr',
  lg: 'px-20pxr py-16pxr text-16pxr'
};

const sizeStyle = sizeStyles[size] 
```